private/pkg/command/command.go

```diff
@@ -30,10 +43,15 @@ import (
 // All external commands in buf MUST use command.Runner instead of
 // exec.Command, exec.CommandContext.
 type Runner interface {
-	// Run runs the external command.
+	// Run runs the external command. It blocks until the command exits.
 	//
 	// This should be used instead of exec.CommandContext(...).Run().
 	Run(ctx context.Context, name string, options ...RunOption) error
+
+	// Start runs the external command, returning a [Process] handle.
+	//
+	// This should be used instead of exec.Command(...).Start().
+	Start(name string, options ...RunOption) (Process, error)
```

Comment:
The options are not specific to Start because all options map to exec.Cmd configuration. That's not just duplication of an option or two but a straight up duplication of the whole command package.

What about renaming RunOption to ExecOption? It's a little clearer and isn't tied to the name of the function.

Reply:
sure sgtm

Reply:
I decided to take it a step further to better reflect what's going on with the difference between Run and Start. It's a but more like exec.Cmd in feel too, which is to say a command is prepared and then ran in two pieces.